amida-tech/saraswati-dashboard
# [SAR-298] and [SAR-343]: Access new HeRA endpoint

# Related Tickets

- [SAR-298](https://jira.amida-tech.com/browse/SAR-298)
- [SAR-343](https://jira.amida-tech.com/browse/SAR-343)

# Other Repos' PR(s) Intended to Work With This PR

- [HeRA-11](https://github.com/amida-tech/saraswati-hedis-results-api/pull/11)

# How Things Worked (or Didn't) Before This PR

HeRA's API is updating it's API, dashboard was using the old endpoint and needed to use the updated one.

# How Things Work Now (And How to Test)

Now the dashboard can search by either member ID or measurement type. It does not use this data in the chart, which is a work in progress still, but you can check that the call worked as intended on the HeRA side and that the dashboard got the correct info while debugging.

To test, stand up the following containers:

1. Hedis Results API
2. MongoDB
3. CQL Execution
4. Dashboard

In CQL Execution run the command `yarn test:seed`. Verify the data was set properly in HeRA, and then refresh the dashboard. Verify HeRA completed the request. [This tutorial](https://code.visualstudio.com/docs/nodejs/reactjs-tutorial) can also be followed to debug dashboard.  Just don't choose Edge, choose Chrome.
